package.json

```diff
@@ -115,13 +125,13 @@
     "text-diff": "^1.0.1",
     "unified": "^9.2.0",
     "unist-util-visit": "^2.0.3",
-    "url-parse": "^1.5.9",
-    "uuid": "^8.3.0",
+    "url-parse": "^1.5.10",
+    "uuid": "^9.0.0",
     "vfile": "^4.2.0"
   },
   "devDependencies": {
     "@faker-js/faker": "^7.6.0",
-    "@axe-core/puppeteer": "^4.1.1",
```

Comment:
It looks like `4.7.0` is available

Reply:
The version above `v4.7.0` requires `node v15.14.0` and above. Although it is not documented, they use a [syntax](https://github.com/dequelabs/axe-core-npm/blob/v4.7.0/packages/puppeteer/src/browser.ts#L64-L67) that is not supported by the current version of node.js, which gives us the error.